### ISSUE TITLE
Placeholder at the top and bottom of the chart

### ISSUE THREAD

**Issue by fxhuhn** [matplotlib/mplfinance#244]
Hi @DanielGoldfarb,

is there a way to set a distance between the top and bottom of the chart so that any signals are not cut off? 

![ice_sy1_de](https://user-images.githubusercontent.com/49436733/90607390-6ecd1480-e201-11ea-931a-28ac0e71b3bc.png)

**Comment by DanielGoldfarb** [comment]
Yes, use kwarg `ylim=( ymin, ymax )` where `( ymin, ymax )` is a two-ple of numbers (int or float).

**Comment by DanielGoldfarb** [comment]
Btw, it appears to me that the chart you posted was made with `tight_layout=True`; if so, that may also be cutting out your signal at the top.   That said, I will look into whether or not addplot data is being considered in determining ymin and ymax.  It may be a little tricky because it is looked at to try to decide whether or not it should be on a `secondary_y`, and _in some cases_ there may be a fine line between choosing a `secondary_y` and expanding the ymin, ymax range.

**Comment by fxhuhn** [comment]
That's right, i'm using thight layout:
`
        mpf.plot(df[['Open','High','Low','Close']],
                 type='candle',
                 volume=False,
                 addplot=add_plots,
                 title=f'{stock_fullname} ({stock_ticker}) - {df.index.max().strftime("%d.%m.%Y")}' ,
                 ylabel='Kurs',
                 style=s,
                 figscale=3.0,
                 figratio=(18, 10),
                 datetime_format = '%d.%m.%y',
                 xrotation = 20,
                 alines=dict(alines=alines,
                             colors='grey',
                             linewidths=1,
                             alpha=0.35),
                 savefig=save,
                 tight_layout=True,
                 closefig=True)
`
It seems that the 'space' of AddPlot are not considered, but only the stock price data.
